.github/actions/cl2-modules/egw/modules/stress-metrics.yaml

```diff
@@ -0,0 +1,70 @@
+steps:
+- name: "{{ .action }} stress metrics"
+  measurements:
+  - Identifier: StressConnections
+    Method: GenericPrometheusQuery
+    Params:
+      action: {{ .action }}
+      metricName: Stress Connections
+      metricVersion: v1
+      unit: count
+      enableViolations: true
+      queries:
+      {{range $test := StringSplit .tests}}
```

Comment:
nit: usually we set parameters in the beginning to see what are the module parameters, like this:
https://github.com/cilium/cilium/blob/2a25a31e23512c0a16b6b73624ad54786c5d147b/.github/actions/cl2-modules/fqdn/modules/profiles.yaml#L1

Reply:
Makes sense, updated.